src/shared/web3/config.ts

```diff
@@ -11,7 +11,9 @@ export const getConfigForChain = (chainId: number): Chain => {
         throw new Error(`Could not find config for chainId ${chainId}`)
     }
 
-    const config: Chain = configEntry[1]
+    // Get a copy of config that is writable
+    const config: Chain = { ...configEntry[1] }
```

Comment:
Subobjects (if there are any) of this copy share memory with the subobjects of the original (shallow copy). Not sure if it's a problem. If it is tho, let's use `deepCopy` or `immer`.